packages/hooks/src/use-event-listener.ts

```diff
@@ -27,11 +27,15 @@ export function useEventListener<K extends keyof DocumentEventMap>(
   React.useEffect(() => {
     const node = runIfFn(env) ?? document
 
+    if (!handler) {
+      return () => {}
```

Comment:
We can safely remove the noop function here:

```suggestion
      return
```